horizons/gui/modules/settings.py

```diff
@@ -225,7 +225,7 @@ def slider_change(self, widget):
 		value = {
 			'volume_music':      lambda x: u'%s%%' % int(500 * x),
 			'volume_effects':    lambda x: u'%s%%' % int(200 * x),
-			'mousesensitivity':  lambda x: u'%+.1f%%' % (200 * x),
+			'mousesensitivity':  lambda x: u'%+.1f' % (x),
```

Comment:
What's the idea here? Previously the shown value would go from 0 to 100%, is the (rather arbitrary) range of 0 to 5 more easily understood by the player?